### PR TITLE
PCA-011(Frequently Asked Questions)

### PR DESCRIPTION
**Pull request by joseph0x**
in this PR, I was able to do the following: 
-created frequently asked questions 
-the link to my ticket https://trello.com/c/wr914kJp/11-pca-011-design-and-develop-responsive-faq-2

### CODE REVIEW

File: src/components/FAQs/FAQ.css [hunk 1]
```diff
@@ -0,0 +1,143 @@
...
+  margin-right: 7.5px;
+}
+#navbar::before {
+  content: "";
+  top: 0;
+  left: 0;
+  height: 226px;
+  width: 100%;
+  position: absolute;
+  background: url(images/bgimage.png);
```
Comment (Qoosim) [inline src/components/FAQs/FAQ.css:46]: - [ ] The path to this image is not correct. It should be like this :point_down: 

```
   background: url(../../images/bgimage.png);
```

File: src/components/FAQs/FAQ.css [hunk 2]
```diff
@@ -0,0 +1,143 @@
...
+  z-index: -3;
+}
+#navbar::after {
+  content: "";
+  top: 0;
+  left: 0;
+  height: 226px;
+  width: 100%;
+  position: absolute;
+  background: url(images/newvector.png);
```
Comment (Qoosim) [inline src/components/FAQs/FAQ.css:60]: - [ ] Samething with this too. Kindly fix it.

File: src/components/FAQs/Nav.js [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+                  Frequently Asked Questions
+                </span>
+              </div>
+            </li>
+          </ul>
+        </div>
+      </div>
+    </nav>
+  );
+}
```
Comment (Qoosim) [inline src/components/FAQs/Nav.js:28]: - [ ] Great job so far. However, there're few things to work on here for your design to match with the figma design. Let's try to create this section with padding left and right to match figma design

### Your design

your-header-part


### Figma design

figma-header-part

File: src/components/FAQs/header.js [hunk 4]
```diff
@@ -0,0 +1,26 @@
...
+import hyphen from "./images/pseudo.png";
+export default function Header() {
+  return (
+    <div className="FAQ--header">
+      <div className="headers">
+        <h6 className="yellow">
+          <img src={hyphen} alt="" />
+          FAQ Questions
+        </h6>
+        <h3 className="black">Get Every General Answers From Here</h3>
```
Comment (Qoosim) [inline src/components/FAQs/header.js]: - [ ] Kindly follow the layout on the figma design to style this heading. You might need to check the font-size use on the original design.

### Your Design
your-gen-ans

### Figma Design
figma-gen-ans

File: src/components/FAQs/header.js [hunk 5]
```diff
@@ -0,0 +1,26 @@
...
+        <div className="header--nav">
+          <a href="" className="bgyellow">
+            General
+          </a>
```
Comment (Qoosim) [inline src/components/FAQs/header.js:16]: - [ ] The border-radius given to this element does not tally with how it appears on the figma design. Kindly check and fix it.

### Your Design
your-btn-shape

### Figma Design
figma-btn-shape

File: src/components/FAQs/Question.js [hunk 6]
```diff
@@ -0,0 +1,34 @@
...
+            className={`plus ${isExpanded ? "hide" : ""}`}
+          />
+          <img
+            src={minus}
+            alt=""
+            className={`plus ${!isExpanded ? "hide" : ""}`}
+          />
+        </div>
+        {isExpanded && <div style={{ padding: "12px" }}>{answer}</div>}
+        <hr />
```
Comment (Qoosim) [inline src/components/FAQs/Question.js:30]: - [ ] Kindly check how this appears on the figma design and fix those areas I pointed out. 
- [ ] Also, check out the thickness and color opacity of the liine

### Your Desgin
your-line-size

### Figma Design
figma-line-size

**Review by Qoosim — CHANGES_REQUESTED** [review]
#  *Status: Required Changes* :recycle:

Hi @joseph0x, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
-  No linter errors. :heavy_check_mark: 
-  Good README file. :heavy_check_mark: 
-  Correct . :heavy_check_mark: 




Cheers and Happy coding!👏👏👏

**Comment by joseph0x** [comment]
changesmadePCA-011(#3)

changesmadePCA-011(#2)

how about now sir?

**Review by joseph0x — COMMENTED** [review]
corrections made

**Pull request merged**